package.json

```diff
@@ -61,7 +61,7 @@
     "@types/request": "^2.0.3",
     "@types/rimraf": "^2.0.2",
     "@types/semver": "^5.3.33",
-    "@types/sinon": "^2.3.5",
```

Comment:
Would it not be better if the type definitions and the package itself has the same version? Last version I could find  is 5.0.1

Reply:
_reset_ It broke the build for some reason.